src/b_setup.hpp

```diff
@@ -106,7 +106,8 @@ void setup()
 
 //Turn on dew heater
 #if DEW_HEATER == 1
-    digitalWrite(DEW_HEATER_PIN, HIGH);
+    digitalWrite(DEW_HEATER_PIN1, HIGH);
+    digitalWrite(DEW_HEATER_PIN2, HIGH);
```

Comment:
While currently harmless, should we support one _or_ two dew heaters?

Reply:
For now I just wanted to get it enabled so it can be used like the other one. In a future update we can add control of each one individually with the option to be on/off by default.